tests/conftest.py

```diff
@@ -1,9 +1,15 @@
-import sys
-import pytest
 import pandas as pd
-from tests.data.snake_oil_data import ensembles_response
 from requests import HTTPError
-from unittest.mock import Mock, MagicMock
+
+from tests.data.snake_oil_data import ensembles_response
+from selenium.webdriver.chrome.options import Options
+
+
+def pytest_setup_options():
+    options = Options()
+    options.add_argument("--headless")
+    options.add_argument("--disable-gpu")
```

Comment:
Some of our components are hardware accelerated, will disable-gpu affect testing those?

Reply:
hmm... i guess we can remove that and only run with `--headless` ? I think that should be sufficient! I'll change and check if the tests are running 👍